v2/datastream-to-spanner/src/test/resources/DataStreamToSpannerEndToEndIT/mysql-schema.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE `Authors` (
+    `author_id` int NOT NULL, -- To: category_id INT64
+    `name` varchar(25),             -- To: full_name STRING(25) Column name renamed
+    `last_update` timestamp,        -- To: Column dropped in spanner
+    PRIMARY KEY (`author_id`)
+);
+
+CREATE TABLE `Books` (
+    `id` int NOT NULL,
+    `title` varchar(200),
+    `author_id` int,
+    FOREIGN KEY (author_id) REFERENCES Author(author_id)
```

Comment:
As mentioned earlier, mysql should not contain the FK constraint.

Reply:
Even if MySql has the FK constraint and we migrate with child rows migrated first the dlq does get created(I checked manually as well, it does). i don't understand why parent should not have FK constraint. What difference does this make?